server/game/cards/02.5-FHNS/CrisisBreaker.js

```diff
@@ -1,10 +1,26 @@
 const DrawCard = require('../../drawcard.js');
 
 class CrisisBreaker extends DrawCard {
-    setupCardAbilities(ability) { // eslint-disable-line no-unused-vars
+    setupCardAbilities() {
+        this.action({
+            title: 'Ready and bring into play',
+            condition: () => this.game.currentConflict && this.game.currentConflict.conflictType === 'military' &&
```

Comment:
This seems unnecessarily complicated - wouldn't something similar to this be simpler? It would only recalculate the conflict skill twice, as opposed to four times, allowing us to get rid of the comparison methods completely:

    () => {
        const currentConflict = this.game.currentConflict;
    
        if (!currentConflict || currentConflict.conflictType !== 'military') {
            return false;
        }
    
        const originalSkillFunction = currentConflict.skillFunction;
    
        currentConflict.skillFunction = (card) => card.getSkill('military');
        currentConflict.calculateSkill();
    
        const conditionFulfilled = currentConflict.attackingPlayer === this.controller ?
            currentConflict.attackerSkill < currentConflict.defenderSkill :
            currentConflict.defenderSkill < currentConflict.attackerSkill;
    
        currentConflict.skillFunction = originalSkillFunction;
        currentConflict.calculateSkill();
    
        return conditionFulfilled;
    }

Reply:
It's definitely cleaner for one card, and if we want to go this way we can. I wrote the other methods under the assumption that this would not be the only card to want to compare strength in a case where that might not be the conflict comparison, and having the functions implemented in conflict prevents us having to do this every time. Again, not opposed to this plan, just trying to future proof. At the very least I could just make the results of compareMilitary it's own value so we don't have to calculate twice